docs/quality-assurance-test-cases.md

```diff
@@ -214,3 +214,14 @@ Only admin users have access to this.
 | Creating a task without a description (either empty or not in the request body at all)    | Fail    |
 | Create a task when a logged user is not involved in the relation                          | Fail    |
 | Create a task if relation state is different than accepted                                | Fail    |
+
+### Confirmation of users email
+
+**Service:** GET /user/confirm_email/{token}
+
+| Test Case                                                                                      | Outcome |
+|------------------------------------------------------------------------------------------------|---------|
+| Verification token entered is one sent on users registered email entered within 24 hrs         | Success |
+| Verification token of already confirmed users account entered                                  | Success |
```

Comment:
Interesting 🤔  If the user is already created the API should not do anything. I guess that counts as a success for now :) @pallavisavant great thought

Reply:
I guess here the expected behavior should be a message with saying the user is already confirmed. All in all, i think this is fine :)